app/src/main/java/com/mymeetings/pairpomodoro/view/MainActivity.kt

```diff
@@ -158,6 +170,14 @@ class MainActivity : AppCompatActivity(), ServiceConnection {
         bindService(Intent(this, PomodoroService::class.java), this, Context.BIND_AUTO_CREATE)
     }
 
+    override fun onResume() {
+        super.onResume()
+        timerProgressBar.max =
+            TimeUnit.MINUTES.toMillis(
+                PreferenceManager.getDefaultSharedPreferences(this).getInt(FOCUS_TIME_KEY, 25).toLong()
```

Comment:
This should not be from sharedPreference.

Reply:
Again they progress bar max was getting reset, will check this scenario with the Android progress bar.

Reply:
This issue is now fixed. We have to call timerCreated method again whenever our activity call service for handshake